### PR TITLE
Angular auth3

### PR DESCRIPTION
**Pull request by jmilleriec**
Merging Conflicts

### CODE REVIEW

File: src/app/components/signup/signup.component.ts [hunk 1]
```diff
@@ -40,7 +40,7 @@ export class SignupComponent extends CommonComponent{
     const newtmpUser: User = { username , password } as User;
     this.signupService.addUser(newtmpUser)
       .subscribe((tmpUser) => {
-
+<<<<<<< HEAD
```
Comment (columbusdevsayshello) [inline src/app/components/signup/signup.component.ts]: This is a problem @jmilleriec

File: src/app/components/signup/signup.component.ts [hunk 2]
```diff
@@ -51,7 +51,9 @@ export class SignupComponent extends CommonComponent{
           console.log('User not created, return code: ' + JSON.stringify(obj));
         }
 
-
+=======
```
Comment (columbusdevsayshello) [inline src/app/components/signup/signup.component.ts]: And this.

File: src/app/components/signup/signup.component.ts [hunk 3]
```diff
@@ -51,7 +51,9 @@ export class SignupComponent extends CommonComponent{
...
+        console.debug("User Created: " + newtmpUser.username);
+>>>>>>> 82301b073acfd3901d51e37bbe48fca6348f3713
```
Comment (columbusdevsayshello) [inline src/app/components/signup/signup.component.ts]: And this.

Comment (jmilleriec) [reply src/app/components/signup/signup.component.ts]: So these are all the things that I spent time fixing, and it looks like it still doesn't like what i did...  I'
m clearly not doing something correctly...

**Review by jmilleriec — COMMENTED** [review]
I'd like to get rid of boom, and thought I did, but it doesn't seem like that wrked...

**Pull request merged**